src/mountutils.cpp

```diff
@@ -22,4 +24,12 @@ NAN_MODULE_INIT(MountUtilsInit) {
   .ToLocalChecked());
 }
 
+
+void MountUtilsLog(std::string string) {
```

Comment:
Does there need to be a distinction between "informational" log messages, and "error" log messages?
(or maybe the distinction is obvious already? \*shrug\*)